Jenkinsfile

```diff
@@ -16,7 +16,7 @@ dockerizedBuildPipeline(
   prepare: {
     githubStatusUpdate('pending')
 
-    sh "docker run --name selenium-chrome -d -p 4444:4444 ${seleniumDockerImage}:${seleniumDockerVersion}"
+    sh "docker run --name selenium-chrome -d -p 4444:4444 -v /dev/shm:/dev/shm ${seleniumDockerImage}:${seleniumDockerVersion}"
```

Comment:
This param is recommended in the docs for this docker image.  It turns out we were able to get by without it when running one test at a time but once I turned up `maxInstances` I got errors without this